game.py

```diff
@@ -0,0 +1,141 @@
+import random
+import string
+
+
+class Game:
+    def __init__(self, size=3, characters=string.ascii_letters, target=100):
+        self.size = size
+        self.characters = characters[:self.size - 1]
```

Comment:
what happens when size > number of ascii characters

Reply:
When size > number of ascii characters, self.characters contains all ascii characters and there is no error. I think I would leave it like this because playing with so many characters is not fun anyway, right?